Clients/tests/Swag4Net.Core.Tests/Program.fs

```diff
@@ -1,11 +1,16 @@
+namespace Swag4Net.Code.Tests
+
 module Program =
   open Expecto
-  open ParsingTests
   
   (**
    * run the test with:
-   * dotnet watch -p Swag4Net.Core.Tests run -f netcoreapp2.1
+   * dotnet watch -p Swag4Net.Core.Tests.fsproj run -f netcoreapp2.1
    *)
   
   let [<EntryPoint>] main args =
-    runTestsWithArgs defaultConfig args tests
+    let mutable result = runTestsWithArgs defaultConfig args Swag4Net.Code.Tests.v2.ParsingTests.tests
```

Comment:
You can remove `mutable` and use shadowing

Reply:
sure but will it sum up total count?